README.md

```diff
@@ -297,6 +297,7 @@ Table of Contents
   * [shrtcode API](https://shrtco.de/docs) - Free URL Shortening API without authorization and no request limits.
   * [SerpApi](https://serpapi.com/) - Real-time search engine scraping API. Returns structured JSON results for Google, Youtube, Bing, Baidu, Walmart and many other engines. Free plan includes 100 successful API calls per month.
   * [Similar Words API](https://word-simi.herokuapp.com/) — An API to find similar words, has vocabulary of about 4Million words.
+  * [Slai](https://slai.io) - Fast prototyping platform for machine learning applications. Build, test, and deploy machine learning apps on serverless GPUs. Developer plan includes 1 hour of free compute per month.
```

Comment:
I don't see any pricing page backing the features, can you link it?
Also 1 hour of free compute per month, very  generous.

Reply:
> Unfortunately quite common to manage abuse

yeah..that's true